docs/contracts.rst

```diff
@@ -467,7 +467,7 @@ The following statements are considered modifying the state:
     }
 
 .. note::
-  ``constant`` is an alias to ``view``.
+  ``constant`` was an alias to ``view``, but this is `deprecated <https://github.com/ethereum/solidity/issues/992#issuecomment-322351715>`_ and has been removed as of version 0.5.0.
```

Comment:
There wasn't a 0.5.0 release yet, so this cannot be past tense. Also please do not include the github comment as it is irrelevant. This is the official documentation, no need to prove the content with links to issues.